Source/EmbeddedBoundary/WarpXFaceInfoBox.H

```diff
@@ -0,0 +1,94 @@
+//
+// Created by Lorenzo Giacomel on 14/04/2021.
```

Comment:
Can you add a tiny usual header here and at similar places please?
Just Copyright year and license SPDX so we can make sure the source code is under one license.

Reply:
It's done